ironfish-cli/src/commands/wallet/multisig/sign.ts

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CurrencyUtils, Transaction } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../command'
+import { RemoteFlags } from '../../../flags'
+
+export class MultiSigSign extends IronfishCommand {
+  static description = 'Aggregate signature shares from participants to sign a transaction'
+  static hidden = true
+
+  static flags = {
+    ...RemoteFlags,
+    account: Flags.string({
+      description: 'The account that created the raw transaction',
+      required: false,
+    }),
```

Comment:
should we have an `f` char like `wallet:send`?

Reply:
Fixed